pkg/indexer/indexer.go

```diff
@@ -286,50 +286,54 @@ func indexLogs(
 		// - There's a stored block
 		// - The event block number is greater than the stored block number
 		// - The check interval has passed
+		skipReorgHandling := false
 		if !reorgInProgress &&
```

Comment:
I have never managed to truly convince myself that this `if` statement is correct. I just glaze over it and say "this is too complicated for me to dig into". Not the best approach, I know. Adding another layer of if statements makes this virtually impossible to reason about. I am hoping that someone other than Borja knows what this does :)